searchlib/src/vespa/searchlib/test/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ vespa_add_library(searchlib_test_gtest_migration OBJECT
     ft_test_app_base.cpp
     initrange.cpp
     searchiteratorverifier.cpp
+    weightedchildrenverifiers.cpp
```

Comment:
How could this work before ... empty ?

Reply:
Undetected ODR violations that could cause missing detection of test failures.